app/jobs/regular/notify_mailing_list_subscribers.rb

```diff
@@ -7,6 +7,17 @@ class NotifyMailingListSubscribers < Jobs::Base
 
     sidekiq_options queue: 'low'
 
+    sidekiq_options retry: 3
+
+    sidekiq_retry_in do |count, exception|
+      case exception.wrapped
+      when SocketError
```

Comment:
Does a `SocketError` always mean it's a temporary issue?

Reply:
In theory, yes. SocketError is quite generic, but I believe most of the times, the socket errors are actually temporary.

I could hard code the exception message from the bug report, but I believe this solution is better.

I also do not think there is no guarantee that even the problem mentioned on Meta is temporary. That could be due to a misconfiguration that will persist.